controllers/udproute_controller.go

```diff
@@ -35,7 +36,9 @@ type UDPRouteReconciler struct {
 	client.Client
 	Scheme *runtime.Scheme
 
-	log logr.Logger
+	log                   logr.Logger
+	ReconcileRequestChan  <-chan event.GenericEvent
```

Comment:
NIT can you rename this to like "ClientReconcileRequestChan" Or somthing? I just want it to be well known that this infra IS NOT provided by operator-sdk :)

Reply:
Sure, done